images/inference-services/kserve/templates/ifs.lib.yaml

```diff
@@ -0,0 +1,17 @@
+#@ def name(inferenceService):
+#@  if inferenceService.appName:
+#@    return inferenceService.appName
+#@  else:
+#@    return inferenceService.project + "-" + inferenceService.codeset + "-" + inferenceService.workflow
+#@  end
+#@ end
+
+#@ def labels(inferenceService):
+fuseml/app-name: #@ name(inferenceService)
+#@ if/end inferenceService.project:
+fuseml/org: #@ inferenceService.project
+#@ if/end inferenceService.codeset and inferenceService.project:
+fuseml/app-guid: #@ inferenceService.project + "." + inferenceService.codeset
```

Comment:
This changes the `app-guid` structure to remove the workflow bit from it. Why ? We could at least settle on a common schema for this label that isn't used anywhere and has not documented purpose.

Reply:
Same here, I was based on the code that did not include the workflow on the app name. I have added workflow to guid.

Thanks.